modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -625,39 +625,51 @@ public void testBuildTemplateFMPortletWithoutBOM() throws Exception {
 	}
 
 	@Test
-	public void testBuildTemplateFormField() throws Exception {
+	public void testBuildTemplateFormField70() throws Exception {
```

Comment:
Thanks @natocesarrego  for updating the tests for both 70 and 71

Reply:
No problem. Actually, I was glad to make it happen! 😉